dv/uvm/icache/data/vcs/vcs.mk

```diff
@@ -0,0 +1,117 @@
+# Copyright lowRISC contributors.
```

Comment:
Remove this file - it is not needed. It s a vestige from the legacy Mekefile flow.